infrastructure/main.tf

```diff
@@ -192,5 +194,9 @@ module "claim-store-api" {
     ASYNC_MAX_THREADPOOL_SIZE = 50
 
     ROOT_APPENDER = "CMC"
+
+    // document management
+    DOCUMENT_MANAGEMENT_URL = "${local.document_management_url}"
+
```

Comment:
you will also need to define the case worker roles here - similar to others `${var.?}`